nav2_safety_nodes/include/nav2_safety_nodes/nav2_safety_node.hpp

```diff
@@ -0,0 +1,154 @@
+#ifndef NAV2_SAFETY_NODES__NAV2_SAFETY_NODE_HPP_
+#define NAV2_SAFETY_NODES__NAV2_SAFETY_NODE_HPP_
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include "rclcpp/rclcpp.hpp"
+#include "visualization_msgs/msg/marker.hpp"
+
+// #include "tf2/convert.h"
+#include "tf2_ros/buffer.h"
+#include "tf2_ros/message_filter.h"
+#include "tf2_ros/transform_listener.h"
+#include "tf2_ros/create_timer_ros.h"
+#include "tf2_sensor_msgs/tf2_sensor_msgs.h"
+
+#include "sensor_msgs/msg/point_cloud2.hpp"
+#include "sensor_msgs/point_cloud2_iterator.hpp"
+#include "laser_geometry/laser_geometry.hpp"
+#include "geometry_msgs/msg/polygon_stamped.hpp"
+#include "geometry_msgs/msg/point.hpp"
+#include "nav2_msgs/msg/speed_limit.hpp"
+#include "nav2_util/lifecycle_node.hpp"
+#include "nav2_util/string_utils.hpp"
+namespace nav2_safety_nodes
```

Comment:
add proper spacing between namespace, includes, and class definitions. See other files for examples.